kong/plugins/moesif/log.lua

```diff
@@ -77,12 +77,16 @@ local function send_payload(sock, parsed_url, batch_events, conf)
 
   if conf.enable_reading_send_event_response then 
     ngx_log(ngx.DEBUG, "[moesif] As reading send event response is enabled, we are reading the response " .. " for pid - ".. ngx.worker.pid())
-    local send_event_response = helper.read_socket_data(sock, conf)
+    local send_event_response, send_event_response_error = helper.read_socket_data(sock, conf)
     if conf.debug then
-      if send_event_response ~= nil then 
-        ngx_log(ngx.DEBUG,"[moesif] send event response - ", send_event_response)
+      if send_event_response_error == nil then 
+        if send_event_response ~= nil then 
+          ngx_log(ngx.DEBUG,"[moesif] send event response - ", send_event_response)
+        else
+          ngx_log(ngx.DEBUG,"[moesif] send event response is nil ")
+        end
       else
-        ngx_log(ngx.DEBUG,"[moesif] send event response is nil ")
+        ngx_log(ngx.DEBUG,"[moesif] error while reading response after sending event - ", send_event_response_error)
```

Comment:
lets print  send_event_response here also

Reply:
also print numOfEvents in this batch here and at line 84

Reply:
okay, added numOfEvents in this batch here in the log